src/test/java/lotto/domain/LottoTest.java

```diff
@@ -15,22 +15,54 @@
 import static org.junit.jupiter.params.provider.Arguments.arguments;
 
 class LottoTest {
-    List<LottoNumber> lottoNumbers;
-    List<LottoNumber> otherLottoNumbers;
-
-    @BeforeEach
-    void setUp() {
-        lottoNumbers = new ArrayList<>(Arrays.asList(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3)));
-        otherLottoNumbers = new ArrayList<>(Arrays.asList(new LottoNumber(2), new LottoNumber(3), new LottoNumber(4)));
-    }
 
     static Stream<Arguments> comparingSource() {
         return Stream.of(
-                arguments(new ArrayList<>(Arrays.asList(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3))),
-                new ArrayList<>(Arrays.asList(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3))), 3),
+                arguments(
+                        new ArrayList<>(Arrays.asList(
+                                new LottoNumber(1)
+                                , new LottoNumber(2)
+                                , new LottoNumber(3)
+                                , new LottoNumber(4)
+                                , new LottoNumber(5)
+                                , new LottoNumber(6)
+                            )
+                        ),
+
+                        new ArrayList<>(Arrays.asList(
+                                new LottoNumber(1)
+                                , new LottoNumber(2)
+                                , new LottoNumber(3)
+                                , new LottoNumber(4)
+                                , new LottoNumber(5)
+                                , new LottoNumber(6)
+                            )
+                        )
```

Comment:
1. List가 두겹이네요!
2. 일반적으로 쉼표 뒤에 줄바꿈을 합니다ㅎㅎ
3. source보다 테스트 메서드를 위에 위치시키는 것이 더 가독성이 좋습니당
```suggestion
                        Arrays.asList(
                                new LottoNumber(4),
                                new LottoNumber(5),
                                new LottoNumber(6),
                                new LottoNumber(7),
                                new LottoNumber(8),
                                new LottoNumber(9)
                        )
```